### PR TITLE
Show prefix "sha256:" while running docker images --no-trunc.

### LINKED ISSUE #1054: docker images --no-trunc missing image id prefix
When executing docker images --no-trunc the user should see the non-truncated output -- this provides the complete image id.  For example in native docker:

```
REPOSITORY          TAG                 IMAGE ID                                                                  CREATED             SIZE
busybox             latest              sha256:47bcc53f74dc94b1920f0b34f6036096526296767650f223433fe65c35f149eb   11 weeks ago        1.113 MB
```

In vic we return:
```
REPOSITORY          TAG                 IMAGE ID                                                           CREATED             SIZE
busybox             latest              47bcc53f74dc94b1920f0b34f6036096526296767650f223433fe65c35f149eb   11 weeks ago        1.113 MB
```

We just need to provide the sha256 prefix.

### PR DESCRIPTION
**Pull request by renmaosheng**
Fixes #1054 
[specific ci=1-03-Docker-Images]

### CODE REVIEW

**Comment by renmaosheng** [comment]
Hi Zach，just notice we already have a test, updated it.

**Comment by hickeng** [comment]
@steven-zou This looks good to me, but the ImageConfig we're changing is one of the elements that serialized into the personality kv store (if I recall correctly).
We should run an upgrade and ensure that the inspect and image listing output is sane afterwards. I don't believe the upgrade test exercises the inspect paths significantly so we might need to manually verify.

File: tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot [hunk 1]
```diff
@@ -61,7 +61,7 @@ No-trunc images
     Should Be True  ${count} >= 3
     @{lines}=  Split To Lines  ${output}
     @{line}=  Split String  @{lines}[2]
-    Length Should Be  @{line}[2]  64
+    Length Should Be  @{line}[2]  71
```
Comment (zjs) [inline tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot]: In addition to this, can we explicitly verify that the prefix is what we expect (`sha256:`)? Or perhaps verify that the value matches a regular expression (e.g., `sha256:[0-9a-f]{64}`)? I'm not sure that the significance of 71 will be clear if this test begins failing in the future.

Comment (renmaosheng) [reply tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot]: done.

**Comment by renmaosheng** [comment]
Hi George, will verify as you suggested. thanks.

**Comment by renmaosheng** [comment]
Hi George, @hickeng 
     After upgrading, the older images' --no-trunc output is still without sha256 prefix. 
> docker -H 10.192.199.135:2375 images --no-trunc
REPOSITORY          TAG                 IMAGE ID                                                                  CREATED             SIZE
busybox             latest              sha256:59788edf1f3e78cd0ebe6ce1446e9d10788225db3dedcfd1a59f764bad2b2690   4 weeks ago         1.15MB
tutum/dnsutils      latest              8f4386247fcf8f247673eeb0ac40a8ac5c5114970ccae133511196553f4e4f16          3 years ago         200MB
do you mean we need to upgrade all the old images during the upgrade process? thanks. 
Best Regards,
Steven

**Comment by hickeng** [comment]
@zjs @renmaosheng 

Options are:
1. inject the type in the returned value and don't serialize it (status quo but fixes the value returned). Benefit is basically zero risk.
2. Hybrid approach where we serialize the type for new images, and inject `sha256` directly if it's missing when returning to user. Over time the images will be updated. Benefit is we don't have to deal with existing images.
3. add code during the load when the endpointVM starts to fix up any image IDs without the prefix. Benefit is that after three minor releases we can remove the fixup code as we don't support upgrade from n-3.

I'm actually in favor of (3) - primarily because it allows for a very focused piece of code that can be removed easily in the future, but still improves on the current state of affairs.

**Pull request closed without merging**